app/src/main/kotlin/com/music/echo/echomusic/updater/downloadmanager/EchoNotificationProvider.kt

```
@@ -44,112 +43,106 @@ class EchoNotificationProvider(
+                Notification.EXTRA_MEDIA_SESSION
+            )?.let {
+                target.extras.putParcelable(Notification.EXTRA_MEDIA_SESSION, it)
+            }
+        }
+    }
 
     private fun setShortCriticalTextSafely(builder: Notification.Builder, text: String) {
         try {
             val method = Notification.Builder::class.java.getMethod("setShortCriticalText", CharSequence::class.java)
             method.invoke(builder, text)
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)